virtcontainers/persist.go

```diff
@@ -61,7 +67,7 @@ func deviceToDeviceState(devices []api.Device) (dss []persistapi.DeviceState) {
 	return
```

Comment:
Although not really part of this PR, you could probably get rid of this return if you feel like it goes with some of the other clean up.

Reply:
Right 👍